SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,66 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2025-02-13T10:10:09.000Z'
+  last-updated: '2024-02-13'
+  last-reviewed: '2024-02-13'
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
```

@adriandiglio can you explain what this commit hash is in reference to? I tried using `git` and GitHub I cannot find it, was/is this based on your fork in GitHub?

---

@xee5ch Argh! Great catch! That was a copy paste error. The spec (https://github.com/ossf/security-insights-spec/blob/v1.0.0/specification.md) says that the commit-hash field is "The last commit to which the SECURITY-INSIGHTS.yml refers." Because this is in the Header, which is supposed to contain "high-level information about the project", I think this commit-hash is supposed to refer to a previous commit to our repo.

---

I am going to review the spec because that leads to interesting problems like this one that I would call "hashception": it is hard to reference a commit hash inside the commit itself references or it changes the commit hash. I guess you might want to make it this then for the time being, but it will point to the latest version after you merged the recommend suggestions below, *not* this commit, but we do what we can, eh?

```suggestion
  commit-hash: bcbdfd5dfe0a105be02dabd503a67c55e2fa3344
```